pkgs/development/libraries/audio/faun/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, libpulseaudio
+, libvorbis
+}:
+
+stdenv.mkDerivation rec {
+  pname = "faun";
+  version = "0.1.4";
+
+  src = fetchFromGitHub {
+    owner = "WickedSmoke";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-XfblM/EKd0fa8CHKpTLUTi7BTj8ASh6v/Mgu6W3RvOI=";
+  };
+
+  buildInputs = [
+    libpulseaudio
+    libvorbis
+  ];
+
+  configurePhase = ''
+    patchShebangs .
```

Comment:
This should be done in postPatch or preConfigure to use the standard configurPhase

Reply:
Unfortunately the standard configurePhase won't work here because it runs `./configure --prefix=$out` which this nonstandard `configure` doesn't like, it wants `./configure --prefix $out` (no equals) :man_shrugging:

Reply:
Do we just need to to patchShebang the configure script or anything else? It is always nice if we can avoid the . for all files.